### DISCUSSION TITLE
Create a saved reading list icon on DEV's profile

### DISCUSSION THREAD

**Discussion by Eveieve** [forem/forem#19609] [Feature Requests]
Hi DEV team, this is my first discussion on Forem. I'll keep the post brief and to the point. 

## Suggestion

**I suggest a saved reading list icon be created on the user's [profile page](https://dev.to/zeyidev)**. This icon should be easy to navigate when the user opens their profile page. The same icon that lets a user save an article should do the job. 

## Because

Currently, a user can only see their saved reading list only when they navigate to their round profile icon and click 'Reading list', which in my opinion, **is too many extra steps for a pretty important feature.** 

As a user, I save articles because I want to read them when I've got more time on my hands, but I often forget I saved this really interesting article and end up not reading them all. When I saved the article, I had the assumption that this saved article list will be readily available on the profile page, but I couldn't find it at first and that led me to abandon my reading list for days. I only realized I can go to the profile icon -> 'Reading list' recently.

**Comment by ghost** [comment]
Hi, I'm Bilal also known as dev sobil
I'm new here, how will I explore

**Comment by jline9507** [comment]
he Reading List button is located on the right side of the Bookmarks Bar[.](https://www.cryptela.com/en/page/crypto-education) If you don't see it, make sure the Bookmarks Bar is visible.
